spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/pulsar.adoc

```diff
@@ -151,7 +151,7 @@ To use `PulsarListener`, you need to use the `@EnablePulsar` annotation.
 When you use Spring Boot support, it automatically enables this annotation and configures all the components necessary for `PulsarListener`, such as the message listener infrastructure (which is responsible for creating the Pulsar consumer).
 `PulsarMessageListenerContainer` uses a `PulsarConsumerFactory` to create and manage the Pulsar consumer.
 
-Spring Boot auto-configuration also provides this consumer factory which you can further configure by specifying any of the {spring-boot-pulsar-config-props}[`spring.pulsar.consumer.*`] application properties.
+Spring Boot auto-configuration also provides this consumer factory which you can further configure by specifying most of the {spring-boot-pulsar-config-props}[`spring.pulsar.consumer.*`] application properties (`subscription-name` is ignored and is instead generated when not specified on the annotation).
```

Comment:
There's actually a typo (that was there before as well) in the property name - it should actually be `spring.pulsar.consumer.subscription.name` (instead of `.subscription-name`).

Reply:
Ahh yes, this is why we do these code review things :) 

This field used to be accurate before the move to Spring Boot where we synthesized the imperative and reactive properties together and simplified them. I will scan the doc in its entirety for any such changes.